bindings/java/src/main/java/ethereum/ckzg4844/Tuple.java

```diff
@@ -1,5 +1,6 @@
 package ethereum.ckzg4844;
 
+/** A tuple holding 2 byte arrays. */
 public class Tuple {
```

Comment:
Do you think it's worth renaming this class? Something like `ByteArrayTuple`?

Initially, this class used generics but I ran into an issue in JNI and switched to hardcoded `byte[]` types. It may be worth revisiting this to see if you get it working with generics.